packages/fxa-content-server/app/scripts/templates/connect_another_device.mustache

```diff
@@ -2,7 +2,7 @@
   <header class="hidden" id="fxa-connect-another-device-header"><h1 class="card-header mb-8">{{#t}}Connect another device{{/t}}</h1></header>
   {{#showSuccessMessage}}
     {{#isSignedIn}}
-      <h2 id="fxa-connected-heading" class="card-header mb-8 ltr:before:content-circle-check rtl:after:content-circle-check rtl:after:relative"><span class="align-top ltr:pl-3 rtl:pr-3">{{#t}}Firefox account connected{{/t}}</span></h2>
+      <h2 id="fxa-connected-heading" class="card-header mb-8 before:content-circle-check"><span class="align-top ltr:pl-3 rtl:pr-3">{{#t}}Firefox account connected{{/t}}</span></h2>
```

Comment:
I realized while working on similar `content-image` additions that you don't actually need `rtl:after` stuff when using `ltr:before`... it's smart enough to flip without it and in fact it _doesn't_ flip when you use `rtl:after`.